src/main/java/com/spotify/google/cloud/pubsub/client/Publisher.java

```diff
@@ -381,6 +444,7 @@ public static Builder builder() {
     private int batchSize = 1000;
     private int concurrency = 64;
     private Listener listener;
+    private long maxLatencyMs = 100;
```

Comment:
What's the rationale for 80ms vs 100ms?

Maybe this should go into the FoS config instead?